libs/datamodel/core/tests/types/negative.rs

```diff
@@ -109,6 +109,29 @@ fn should_fail_on_custom_related_types() {
     ));
 }
 
+#[test]
+fn should_fail_on_native_type_with_invalid_datasource_name() {
+    let dml = r#"
+        datasource db {
+          provider = "postgres"
```

Comment:
```suggestion
          provider = "postgresql"
```
Ooo!, just missed this one

Reply:
@rahul3v : Both `postgres` and `postgresql` are ok to use as a provider argument. Our engineers usually call it postgres when talking and hence bias to use it as provider as well i guess.

Reply:
if that's correct then good to go...!😃